### DISCUSSION TITLE
iOS使用PAGText解析👩🏼‍❤️‍👨🏽emotion表情分割有问题，会解析成多个

### DISCUSSION THREAD

**Discussion by loveAct** [Tencent/libpag#2584] [Q&A] [answered]
下面使用demo的bulgetext素材，获取PAGText，替换👩🏼‍❤️‍👨🏽👌表情，会发现生成的表情被分割成2个小表情，应该分割代码没分割好。望哪位大神帮忙解决一下
`NSString* path = [[NSBundle mainBundle] pathForResource:@"bulgetext" ofType:@"pag"];
        PAGFile* pagFile = [PAGFile Load:path];
        if ([pagFile numTexts] > 0) {
            PAGText* textData = [pagFile getTextData:0];
            textData.text = @"👩🏼‍❤️‍👨🏽👌";
            [pagFile replaceText:0 data:textData];
        }
`

![20241113-173346](https://github.com/user-attachments/assets/a0d7dd41-3f36-48e0-81f3-59228a1c4673)

**Comment by domchen** [comment, marked as answer]
感谢反馈，这个问题已经在 #2633 里修复，等最近下次再发个版本带出去。

**Comment by kevingpqi123** [comment]
可以升级至 4.4.20 版本
